docs/automations/integrations/LinearBAI/describe-pr/README.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Automation - Add PR Description Using LinearB's AI
+description: Use gitStream's integration with AI services to generate a summary of the changes in the PR.
+category: [quality, genai, copilot, tests, efficiency]
+starter_kits: [genai]
+---
+# Add PR Description Using LinearB's AI
+
+<!-- --8<-- [start:example]-->
+Use the `AI_DescribePR` filter to automatically generate and append a concise, AI-generated description to a pull request. This ensures that all PRs include meaningful and helpful descriptions, improving review efficiency.
+
+![summarized-pr](/automations/integrations/askAI/summarize-pr/LinearB-AI-describe-pr.png)
```

Comment:
the image is outdated, please add a new one
https://github.com/linear-b/gitstream-rules-resolver/pull/61
https://github.com/linear-b/linpr/pull/694